export/export_pipeline/compute_gltf_representation.py

```diff
@@ -0,0 +1,270 @@
+  selected_actors = unreal.Set(unreal.Actor)
+  selected_actors.add(actor)
+  print(f'Writing actor {name} to {filename}...')
+  count = 0
+  N = 10
+  while not unreal.GLTFExporter.export_to_gltf(None, filename, options, selected_actors):
+  # while not unreal.Exporter.run_asset_export_task(task):
+    print(f'Retry {count+1} / {N}')
+    time.sleep(0.5)
+    if count == 10:
+      break
+  else:
+    return True
+  return False
+
+
+PhysicsConstraint = namedtuple(
+  'PhysicsConstraint',
+  ['parent', 'child', 'name', 'range', 'ref', 'axis', 'type']
+)
+MeshComponent = namedtuple('MeshComponent', ['decompose_method', 'decompose_group'])
+
+
+if __name__ == '__main__':
+  parser = argparse.ArgumentParser()
+  parser.add_argument('--scene_path', required=True)
+  parser.add_argument('--actors', default=None,
+                      help='Comma separate list of actor names - all others will be excluded if this option is used')
+  args = parser.parse_args()
+  filename = osp.join(osp.dirname(osp.abspath(__file__)), 'params.yaml')
+  with open(filename, 'r') as f:
+    params = yaml.safe_load(f)
+  # TODO(samarth): access params stuff by attributes
+  export_dir = osp.expanduser(osp.join(args.scene_path, params['common']['GLTF_SCENE_DIR_NAME']))
+  os.makedirs(export_dir, exist_ok=True)
+  include_actors = args.actors.split(',') if args.actors else None
+  
+  filename = osp.join(export_dir, 'actors_information.json')
+  if osp.isfile(filename):
+    with open(filename, 'r') as f:
+      body_properties = json.load(f)
+  else:
+    body_properties = {}
+  
+  # Iterate through the actors
+  for actor in unreal.GameplayStatics.get_all_actors_of_class(world, unreal.Actor):
+    if isinstance(actor, unreal.CameraActor) or isinstance(actor, unreal.SceneCapture2D) or \
+      isinstance(actor, unreal.PointLight):
+      continue
+    actor_name = actor.get_actor_label()
+    if 'HDRIBackdrop' in actor_name:
+      continue
+    if include_actors and (actor_name not in include_actors):
+      continue
+    
+    this_body_properties = {'geoms': {}, }
+    # check if actor has geometry
+    has_geometry = False
+    i = 0
+    for c in actor.get_components_by_class(unreal.StaticMeshComponent):
+      static_mesh = c.get_editor_property('static_mesh')
+      if static_mesh is None:
+        print(f'actor {actor_name} component {c.get_name()} does not have a static mesh')
+      elif static_mesh.get_name() != 'SM_Dummy':
+        # TODO(samarth) get MeshComponent elements from UE component
+        c_name = c.get_name()
+        if c_name == 'StaticMeshComponent0':
+          c_name = actor_name
+        this_body_properties['geoms'][c_name] = MeshComponent('coacd', i)._asdict()
+        has_geometry = True
+        i += 1
+
+    if has_geometry:
+      export_meshes(actor, export_dir)
+    else:
+      print(f'Actor {actor_name}, does not have geometry')
+      continue
+    
+    if isinstance(actor, unreal.StaticMeshActor):
```

Comment:
- I don't think this logic is correct, `simulate_physics` is really a per-component property, so we probably should not try to collapse it to a per-actor property here.
- Let's call the property that we're saving `simulate_physics` because it's good to match the existing Unreal name unless we have a good reason to deviate.

Reply:
renamed `moving` to `simulate_physics`, need to discuss about the other point. It is mentioned as the unchecked item in README.md.